### ISSUE TITLE
Selectors not working

### ISSUE THREAD

**Issue by AlexandrOsiptsov** [Arustinal/fluentogram#22]
## Environment

- **OS**: MacOS Ventura 13.0.1
- **Python**: 3.11.2
- **aiogram**: 3.4.1
- **aiogram-dialog**: 2.1.0
- **fluentogram**: 1.1.6
- **fluent-compiler**: 0.3
- **fluent.syntax**: 0.19.0

## Problem

Following code changes nothing in the stub-file after the command `i18n -ftl _locale-path_/txt.ftl -stub _stub-path_/stub.pyi` has been run:

```fluent
## txt.ftl

emails =
    { $unreadEmails ->
        [one] You have one unread email.
       *[other] You have { $unreadEmails } unread emails.
    }
```

Also, following code after running similar command makes invalid stub-file:

```fluent
## txt.ftl

emails-some-more =
    { $unreadEmails ->
        [one] You have one unread email.
       *[other] You have { $unreadEmails } unread emails.
    }
```

```py
## stub.pyi

from typing import Literal

 
class TranslatorRunner:
    def get(self, path: str, **kwargs) -> str: ...
    
    emails: Emails


class Emails:
    some: EmailsSome


class EmailsSome:

```

**Comment by Arustinal** [comment]
try fluentogram 1.1.7

**Comment by AlexandrOsiptsov** [comment]
Big thanks! Problem really fixed in `1.1.7`. It works correct, but stub-file seems little bit strange (misalignment and multilines):

```fluent
## txt.ftl

apples =
    { $count ->
        [one] У вас есть { $count } яблоко.
        [few] У вас есть { $count } яблока.
       *[other] У вас есть { $count } яблок.
    }
```

```python
## stub.pyi

from typing import Literal

    
class TranslatorRunner:
    def get(self, path: str, **kwargs) -> str: ...
    
    @staticmethod
    def apples(*, count) -> Literal["""{ $count -&gt;
[one] У вас есть одно яблоко.
[few] У вас есть 
[few] { $count }
[few]  яблока.
*[other] У вас есть 
*[other] { $count }
*[other]  яблок.
}"""]: ...
```
